static_src/test/unit/stores/user_store.spec.js

```diff
@@ -433,6 +433,73 @@ describe('UserStore', function () {
     });
   });
 
+  describe('on CURRENT_UAA_INFO_RECEIVED', function () {
+    it('should not fail when groups is missing in uaaInfo', function () {
+      const currentUaaInfo = { guid: '1234' };
```

Comment:
These tests in each `it` block seem to have repeated information. Is it possible to move the data to a `beforeEach` like you did below?